oneview_redfish_toolkit/blueprints/computer_system.py

```diff
@@ -185,3 +186,50 @@ def change_power_state(uuid):
         # In case of error log exception and abort
         logging.exception('Unexpected error: {}'.format(e))
         abort(status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+@computer_system.route(
+    "/redfish/v1/Systems/<uuid>", methods=["DELETE"])
+def remove_computer_system(uuid):
+    """Removes a specific System
+
+        Args:
+            uuid: The System ID.
+
+        Exceptions:
+            HPOneViewException: When some OneView resource was not found.
+            return abort(404)
+
+            Exception: Unexpected error.
+            return abort(500)
+    """
+    try:
+        # Deletes server profile for given UUID
+        response = g.oneview_client.server_profiles.delete(uuid)
+
+        if response is True:
+            return Response(status=status.HTTP_204_NO_CONTENT)
+
+        # Check if returned a task
+        if type(response) is dict:
+            # Check if task is completed
+            if response['taskState'] == 'Completed':
+                return Response(status=status.HTTP_204_NO_CONTENT)
+
+            # Log task error message if it has one
+            if response['taskState'] in TASK_ERROR_STATES:
+                if 'taskErrors' in response and \
+                    len(response['taskErrors']) > 0:
+                    err = response['taskErrors'][0]
```

Comment:
Can we assume that just a single entry "[0]" will be there? If we could have more, can we assume that the index "0" is our target?

Reply:
That is a good point, I've look into python sdk to check how it handle it, and it get just the first entry to get the error message: 
https://github.com/HewlettPackard/python-hpOneView/blob/master/hpOneView/resources/task_monitor.py#L140

But maybe we can concat with other entries.